tests/test_flintrock.py

```diff
@@ -9,7 +9,12 @@
     option_name_to_variable_name,
     variable_name_to_option_name,
     option_requires,
-    mutually_exclusive
+    mutually_exclusive,
+    get_last_commit_sha
+)
+
+from urllib.error import (
```

Comment:
Nit: I generally reserve `from ... import` for Flintrock objects. Exceptions include datatypes like `namedtuple` and `datetime`.